apps/src/aichat/views/modelCustomization/PromptCustomization.tsx

```diff
@@ -35,6 +43,33 @@ const PromptCustomization: React.FunctionComponent = () => {
   return (
     <div className={styles.verticalFlexContainer}>
       <div>
+        <div className={classNames(styles.inputContainer)}>
```

Comment:
Would a subrender function make sense here for easier readability?

Reply:
Do you mean for all of the new stuff (dropdown/button/dialog)? Just tried something along those lines, LMK what you think. Could probably break up / refactor the rest of this as well, Sanchit and I chatted about reusing some of the components he added on the levelbuilder side (eg, `FieldSection`)

Reply:
> Do you mean for all of the new stuff (dropdown/button/dialog)? Just tried something along those lines, LMK what you think.

Yes - thanks! Look forward to seeing the commit.

> Could probably break up / refactor the rest of this as well

Nice!

> Sanchit and I chatted about reusing some of the components he added on the levelbuilder side (eg, FieldSection)

Looking at Sanchit's`ModelCardFields` and `FieldSection`, they seem different enough from the user view so that making them generic seems like it may take a bit of work. But think definitely work a refactor/using same components after the P0 is reached!

Reply:
hah sorry forgot to push, done!

Reply:
Looks good!